tests/ui/lint/unused/unused-attr-duplicate.stderr

```diff
@@ -289,5 +277,18 @@ note: attribute also specified here
 LL | #[cold]
    | ^^^^^^^
 
+error: unused attribute
+  --> $DIR/unused-attr-duplicate.rs:102:1
+   |
+LL | #[used]
+   | ^^^^^^^ help: remove this attribute
+   |
+note: attribute also specified here
+  --> $DIR/unused-attr-duplicate.rs:101:1
+   |
+LL | #[used]
+   | ^^^^^^^
+   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
```

Comment:
This warning is new, but I think this is reasonable in this context.
Is adding this warning acceptable?

Reply:
Will be reverted until https://github.com/rust-lang/rust/issues/142836 is decided